Sources/Instrumentation/URLSession/URLSessionInstrumentation.swift

```diff
@@ -592,9 +601,10 @@ public class URLSessionInstrumentation {
 
     private func urlSessionTaskWillResume(_ task: URLSessionTask) {
         // AV Asset Tasks cannot be auto instrumented, they dont include request attributes, skip them
-        if let avAssetTaskClass = Self.avAssetDownloadTask,
-           task.isKind(of: avAssetTaskClass) {
-            return
+        for aClass in Self.AVTaskClassList {
```

Comment:
`contains(where:)` would be more idiomatic.

`guard !Self.AVTaskClassList.contains(where: { task.isKind(of: $0) }) else { return }`